examples/getting-started-app/app/roles/storage.lua

```diff
@@ -135,27 +135,27 @@ local exported_functions = {
     customer_update_balance = customer_update_balance,
 }
 
-local function init(opts)
-    if opts.is_master then
+local function init()
+    for name, func in pairs(exported_functions) do
+        rawset(_G, name, func)
+    end
+end
+
+local function apply_config(_, options)
+    if options.is_master then
         init_spaces()
 
         for name in pairs(exported_functions) do
             box.schema.func.create(name, {if_not_exists = true})
             box.schema.role.grant('public', 'execute', 'function', name, {if_not_exists = true})
         end
     end
-
-    for name, func in pairs(exported_functions) do
-        rawset(_G, name, func)
-    end
-
-    return true
```

Comment:
In the Cartridge application template both `init` and `apply_config` functions return `true`. 
This role is an example based on template, I think it should use the same convention.

The same is about function args, I suggest using `local function init(opts) -- luacheck: no unused args`, as it's defined in the template.

Reply:
Return values of this functions are not used

Reply:
OK, I will remove return values from template, but renaming `opts` -> `options` confuses user. Example application is an edited template application, I don't see any reason to rename this parameter.